### DISCUSSION TITLE
Is there a way to use Sprinkles to generate atomic classes with !important?

### DISCUSSION THREAD

**Discussion by tahmid-stacker** [vanilla-extract-css/vanilla-extract#930] [Q&A]
The question is fairly self-explanatory, but I would like to generate class names that use the `!important` keyword. Couldn't find anything in the docs, unless I missed something obvious. Thanks for any help.

**Comment by bjgrosse** [comment]
Would be great if this could just be an option configurable on the createSprinkles function.
